components/project-list/project-list.jsx

```diff
@@ -1,113 +1,49 @@
 import React from 'react';
```

Comment:
I refactored this into 2 components:

- The new `<ProjectList>` component: (`components/project-list/project-list.jsx`)
  - Render things based on what's been passed to it.

- `<ProjectLoader>` component: (`components/project-loader/project-loader.jsx`)
  - Fetches data and pass it to `<ProjectList`> for rendering UI.